Cargo.lock

```diff
@@ -2301,7 +2323,7 @@ dependencies = [
  "tokio",
  "tokio-stream",
  "tokio-tar",
- "tokio-tungstenite",
+ "tokio-tungstenite 0.14.0",
```

Comment:
Let me know if I should update trunk to use tokio-tungstenite 0.15. That is the version axum uses.